subsys/bluetooth/controller/hci/hci.c

```diff
@@ -1557,12 +1558,24 @@ static void vs_read_version_info(struct net_buf *buf, struct net_buf **evt)
 	rp = cmd_complete(evt, sizeof(*rp));
 
 	rp->status = 0x00;
+#if defined(CONFIG_SOC_FAMILY_NRF5)
```

Comment:
Please have a file private macro up in the file that you assign here. The C code will look cleaner here.

Reply:
Can you be more explicit? I am not sure what you are after, a macro for nRF5?

Reply:
in the hci_vs.h  you can have:

#if defined(CONFIG_SOC_SERIES_NRF51)
#define BT_HCI_VS_HW_VAR BT_HCI_VS_HW_VAR_NORDIC_NRF51X
#elif ....
#define BT_HCI_VS_HW_VAR BT_HCI_VS_HW_VAR_NORDIC_NRF5XY

etc...

and use those one each here.

rp->hw_variant = BT_HCI_VS_HW_VAR;